status/status_history.go

```diff
@@ -185,3 +191,16 @@ func (k HistoryKind) Valid() bool {
 	}
 	return false
 }
+
+// AllHistoryKind will return all valid HistoryKinds.
+func AllHistoryKind() map[HistoryKind]string {
+	return map[HistoryKind]string{
+		KindUnit:              "statuses for specified unit and its workload",
+		KindUnitAgent:         "statuses from the agent that is managing a unit",
+		KindWorkload:          "statuses for unit's workload",
+		KindMachineInstance:   "statuses that occur due to provisioning of a machine",
+		KindMachine:           "status of the agent that is managing a machine",
+		KindContainerInstance: "statuses from the agent that is managing containers",
+		KindContainer:         "statuses from the containers only and not their host machines",
```

Comment:
I do wonder why we have the container kinds. shouldn't those just be another machine? Anyway, I don't have anything amazingly better to say. Maybe if I see a paste of what it all looks like when formatted and put together.

Reply:
The final overall look is here - http://pastebin.ubuntu.com/25865227/